### PR TITLE
Cheetah Node.js - v1.1

### PR DESCRIPTION
**Pull request by mrrostam**

### CODE REVIEW

File: binding/nodejs/src/cheetah.ts [hunk 1]
```diff
@@ -48,17 +52,17 @@ export default class Cheetah {
...
+   * to disable endpoint detection.
+   * @param {boolean} options.enableAutomaticPunctuation Flag to enable automatic punctuation insertion.
    */
   constructor(
     accessKey: string,
-    endpointDurationSec: number,
-    modelPath?: string,
-    libraryPath?: string
-  ) {
+    options: CheetahConfig = {}) {
```
Comment (ksyeo1010) [inline binding/nodejs/src/cheetah.ts:65]: can we run formatter in cheetah as well?

File: binding/nodejs/src/cheetah.ts [hunk 2]
```diff
@@ -68,24 +72,18 @@ export default class Cheetah {
       throw new CheetahInvalidArgumentError(`No AccessKey provided to Cheetah`);
     }
 
+    const {
+      modelPath = path.resolve(__dirname, DEFAULT_MODEL_PATH),
+      libraryPath = getSystemLibraryPath(),
+      endpointDurationSec = 1.0,
+      enableAutomaticPunctuation = false} = options;
```
Comment (ksyeo1010) [inline binding/nodejs/src/cheetah.ts:79]:
```suggestion
      enableAutomaticPunctuation = false
    } = options;
```

File: binding/nodejs/src/cheetah.ts (hunk 1 above)
Comment (mrrostam) [reply binding/nodejs/src/cheetah.ts:65]: Actually I did, i guess WebStorm formatter follows a bit different style here

Comment (ksyeo1010) [reply binding/nodejs/src/cheetah.ts:65]: oh ok

**Review by ksyeo1010 — APPROVED** [review]

**Review by laves — APPROVED** [review]

**Pull request merged**